project/interactivemap/settings.py

```diff
@@ -162,8 +162,10 @@
     'interactivemap-frontend-*.now.sh',
 )
 
-AUTH0_DOMAIN = os.environ.get('AUTH0_DOMAIN')
-API_IDENTIFIER = os.environ.get('API_IDENTIFIER')
+AUTH0_DOMAIN = os.environ.get('AUTH0_DOMAIN', 'chronoscio.auth0.com')
+API_IDENTIFIER = os.environ.get('API_IDENTIFIER', 'https://chronoscio.org/')
+AUTH0_CLIENT_ID = os.environ.get('AUTH0_CLIENT_ID', '')
```

Comment:
So I've been able to get it running locally, but tests are failing because I'm missing the correct Auth0 tokens. Can we come up with a way to run tests locally without needing to expose our secret tokens?

Reply:
You should be able to register on Auth0 and input all information from a new application. I can DM you our tokens if you want.

Reply:
Thanks @whirish, I was able to get it working with a new application after setting up Auth0